sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClientLiveTests.cs

```diff
@@ -86,6 +123,10 @@ public async Task CreateMetricAnomalyDetectionConfiguration()
 
             Assert.That(getConfig.Id, Is.EqualTo(createdConfiguration.Id));
 
+            getConfig.Description = "updated";
+
+            await adminClient.UpdateMetricAnomalyDetectionConfigurationAsync(getConfig.Id, getConfig);
```

Comment:
I think it would be worth adding an `Update` test using a user-instantiated instance as well. The one we get from `Get` has lots of fields populated, and we could be missing some null checks.

Same reasoning applies for alert configuration.